src/Entities/Transaction.php

```diff
@@ -725,4 +725,100 @@ public function getReferenceData()
     {
         return $this->getAttribute('referenceData');
     }
+
+    /**
+     * @return bool
+     */
+    public function getIsRebill()
+    {
+        return $this->getAttribute('isRebill');
+    }
+
+    /**
+     * @return string
+     */
+    public function get3ds()
+    {
+        return $this->getAttribute('3ds');
+    }
+
+    /**
+     * @return string
+     */
+    public function getDcc()
+    {
+        return $this->getAttribute('dcc');
```

Comment:
That doesn't correct, it should be a object

Reply:
Removed this getter.